README.md

```diff
@@ -51,6 +51,13 @@ providers:
     # Resource Group name:
     resource_group: 'TestResource1'
     # All are required to authenticate.
+    # Azure RetryPolicy Settings
+    # https://azuresdkdocs.blob.core.windows.net/$web/python/azure-core/1.9.0/azure.core.pipeline.policies.html?highlight=retrypolicy#azure.core.pipeline.policies.RetryPolicy
+    # Total_retries default 10
+    client_total_retries: 10
+    # status_retries default 3
+    client_status_retries: 3
```

Comment:
Super minor, but the convention for whatever reason has been to comment out optional stuff in these.

```suggestion
    #client_total_retries: 10
    # status_retries default 3
    #client_status_retries: 3
```